mkdocs_click/_docs.py

```diff
@@ -100,19 +106,59 @@ def _make_usage(ctx: click.Context) -> Iterator[str]:
     yield ""
 
 
-def _make_options(ctx: click.Context) -> Iterator[str]:
+def _make_options(ctx: click.Context, style: str = "plain") -> Iterator[str]:
```

Comment:
(Nit)
Instead of modifying the entire body of this function, could we consider defining several functions and turning this `_make_options` in one that basically does the `if` switch? That's what I originally meant in my other comment.

```python
def _make_options(...):
    if style == "plain":
        return _make_plain_options(...)
    elif style == "table":
        return _make_table_options(...)
    else:
        raise MkDocsClickException(...)
```